cumulusci/core/config/OrgConfig.py

```diff
@@ -178,3 +180,27 @@ def get_community_info(self, community_name, force_refresh=False):
             )
 
         return self._community_info_cache[community_name]
+
+    @property
+    def installed_packages(self):
+        if not self._installed_packages:
+            response = self.salesforce_client.restful(
```

Comment:
While we're in here, can we update salesforce_client to use the org's API version rather than being hardcoded to 45.0?